app/models/manageiq/providers/hawkular/middleware_manager.rb

```diff
@@ -0,0 +1,63 @@
+module ManageIQ::Providers
+  class Hawkular::MiddlewareManager < ManageIQ::Providers::MiddlewareManager
+    require_nested :RefreshParser
+    require_nested :RefreshWorker
+    require_nested :Refresher
+
+    include AuthenticationMixin
+
+    DEFAULT_PORT = 80
+    default_value_for :port, DEFAULT_PORT
+    has_many :middleware_servers, :foreign_key => :ems_id, :dependent => :destroy
+    has_many :middleware_deployments, :foreign_key => :ems_id, :dependent => :destroy
+
+    def verify_credentials(auth_type = nil, _options = {})
+      auth_type || 'default'
+      connect(_options).list_feeds
```

Comment:
Well, I tested it in the UI and it worked :-) But then that may have be pure magic